cli/cmd/tables_v3.go

```diff
@@ -3,6 +3,7 @@ package cmd
 import (
 	"context"
 	"fmt"
+	gosync "sync"
```

Comment:
```suggestion
	"sync"
```

Reply:
We already have a package named `sync` that's the reason for the different import name

Reply:
Here https://github.com/cloudquery/cloudquery/blob/cc30fb09d1a039948754cbda09a1adf1b58d0c54/cli/cmd/sync.go#L76